packages/monitoring-system/metrics-dashboard/cloudbuild.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: gcr.io/cloud-builders/gcloud
+    id: "get-firebase-token"
+    waitFor: ["-"]
+    entrypoint: 'bash'
+    args: 
+      - '-c'
+      - "gcloud secrets versions access latest --secret=firebase-token-metrics-dashboard --format='get(payload.data)' | tr '_-' '/+' | base64 -d > firebase-token.txt"
+
+  - name: gcr.io/cloud-builders/npm
+    id: "deploy-dashboard"
+    waitFor: ["get-firebase-token"]
+    entrypoint: bash
+    args:
+      - "-e"
+      - "./scripts/deploy.sh firebase-token.txt"
```

Comment:
This assumes that the `pwd` for the Cloud Build is the directory where the `cloudbuild.yml` file is. Not sure if this assumption is correct?

Reply:
the default working directory is `/workspace` which is the root of the repository (for triggered cloud builds). You could provide a `dir` option: https://cloud.google.com/cloud-build/docs/build-config#dir